apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/RefineModal/steps/2_SelectEvaluationResults/SelectableEvaluationResultsTable.tsx

```diff
@@ -29,17 +29,27 @@ export const ResultCellContent = ({
   evaluation: EvaluationDto
   value: unknown
 }) => {
-  if (evaluation.configuration.type === EvaluationResultableType.Boolean) {
+  if (
+    (evaluation.configuration ?? evaluation.metadata.configuration).type ===
+    EvaluationResultableType.Boolean
+  ) {
     return (
       <Text.H4 color={(value as boolean) ? 'success' : 'destructive'}>
         {String(value)}
       </Text.H4>
     )
   }
 
-  if (evaluation.configuration.type === EvaluationResultableType.Number) {
-    const minValue = evaluation.configuration.detail?.range.from ?? 0
-    const maxValue = evaluation.configuration.detail?.range.to ?? 10
+  if (
+    (evaluation.configuration ?? evaluation.metadata.configuration).type ===
+    EvaluationResultableType.Number
+  ) {
+    const minValue =
+      (evaluation.configuration ?? evaluation.metadata.configuration).detail
+        ?.range.from ?? 0
+    const maxValue =
+      (evaluation.configuration ?? evaluation.metadata.configuration).detail
+        ?.range.to ?? 10
```

Comment:
you could dry a bit this code assigning to constants
``` javascript
const configuration = evaluation.configuration
const config = configuration ?? valuation.metadata.configuration

if (config.type ===  EvaluationResultableType.Number) {
  const minValue = config.range.from ?? 00
  const maxValue = config.range.to ?? 10
}
```

Reply:
This check is just for the transition PR. It will be removed in the next PR, so I don't think it is needed